service/windows/installer/resources/licenses.html

```diff
@@ -25784,7 +25784,7 @@ <h3>Apache-2.0</h3>
 See the License for the specific language governing permissions and
 limitations under the License.
 </pre>
-<h2><a href="https://hackage.haskell.org/package/ghc-9.0.2">ghc</a>
+<h2><a href="https://hackage.haskell.org/package/ghc-9.2.5">ghc</a>

```

Comment:
Do not touch the licenses please. They are generated and the text of the license likely changed as well.